hyperspy/io_plugins/fei.py

```diff
@@ -503,17 +537,21 @@ def ser_reader(filename, objects=None, verbose=False, *args, **kwds):
                             'Dim-%i_CalibrationOffset' % (i + 1)][0],
                         'scale': header[
                             'Dim-%i_CalibrationDelta' % (i + 1)][0],
-                        'units': header['Dim-%i_Units' % (i + 1)][0],
+                        'units': header['Dim-%i_Units' % (i + 1)][0] if header['Dim-%i_UnitsLength'%(i+1)]>0 else 'a.u.',
```

Comment:
I had the impression that the units were always in SI unless specified. Is that true in your experience? If yes, coult 'a.u.' be replaced with 'meters'?